lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -356,11 +356,11 @@ def setUp(self):
         # course is outside the context manager that is verifying the number of queries,
         # and with split mongo, that method ends up querying disabled_xblocks (which is then
         # cached and hence not queried as part of call_single_thread).
-        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 15, 3),
-        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 15, 3),
+        (ModuleStoreEnum.Type.mongo, 1, 5, 3, 13, 1),
```

Comment:
@robrap @andy-armstrong Don't you guys think validating number of mongo queries is a little bit too much (i.e. too fragile)? No need to address this comment in scope of this PR of course :)